syne_tune/optimizer/schedulers/hyperband_promotion.py

```diff
@@ -76,7 +75,7 @@ def _find_promotable_config(self, recorded, prom_quant, resource):
 
         :param recorded: Dict to scan
         :param prom_quant: Quantile for promotion
-        :param resource: Amount of resources spent on the rung.
+        :param resource: Rung level
```

Comment:
I found the old comment more expressive. It was removed since it is actually the amount of resources spent minus the resources spent earlier?

Reply:
OK, can bring it back.